apps/ui/src/components/ButtonFollow.vue

```diff
@@ -15,14 +13,18 @@ const followedSpacesStore = useFollowedSpacesStore();
 const { isWhiteLabel } = useWhiteLabel();
 const uiStore = useUiStore();
 
+const spaceIdComposite = computed(
+  () => `${props.space.network}:${props.space.id}`
+);
+
```

Comment:
It used to work before because in our case `space` was reactive object under the hood.